### ISSUE TITLE
declaration of 'unit' in radialBar graph

### ISSUE THREAD

**Issue by Hakzel** [RomRider/apexcharts-card#67]
I am a huge fan of your apexchart-card and I want to switch all my old standard gauges and graphs over to your implementation.

So far I succeeded with most of the cards but I am struggling with building a gauge for anything other than percentages.

For percentages they look perfectly fine:
![image](https://user-images.githubusercontent.com/44771922/107241222-1138b880-6a2b-11eb-9ccf-3ccddc2bf212.png)

But when it comes to gauges for temperatures they are not so handy because of the forced 'percentage sign' after the value.

I found out, that you can change the unit in the radialBar apex options:
dataLabels -> values -> formatter 
But it only accepts javascript which is quite difficult to add there (at least for me)

Maybe you can implement the 'unit' keyword in the yaml so we can specify the unit in there.
This would make the gauges so much more useful :-)

Thanks in advance and keep up the good work!

**Comment by RomRider** [comment]
The thing is that radialBar in ApexCharts only supports percentages unfortunately. :(
There's nothing I can do about this.

**Comment by Hakzel** [comment]
Regarding the documentation the function under
dataLabels -> values -> formatter   

just adds the percentage symbol to the value:

`formatter: function (val) {return val + '%'};`

So it should be possible to swap the '%' to any other string you like. Or am I getting it wrong?

`plotOptions:{
      radialBar: {
          inverseOrder: false,
          startAngle: 0,
          endAngle: 360,
          offsetX: 0,
          offsetY: 0,
          hollow: {
              margin: 5,
              size: '50%',
              background: 'transparent',
              image: undefined,
              imageWidth: 150,
              imageHeight: 150,
              imageOffsetX: 0,
              imageOffsetY: 0,
              imageClipped: true,
              position: 'front',
              dropShadow: {
                enabled: false,
                top: 0,
                left: 0,
                blur: 3,
                opacity: 0.5
              }
          },
          track: {
              show: true,
              startAngle: undefined,
              endAngle: undefined,
              background: '#f2f2f2',
              strokeWidth: '97%',
              opacity: 1,
              margin: 5, 
              dropShadow: {
                  enabled: false,
                  top: 0,
                  left: 0,
                  blur: 3,
                  opacity: 0.5
              }
          },
          dataLabels: {
              show: true,
              name: {
                  show: true,
                  fontSize: '16px',
                  fontFamily: undefined,
                  fontWeight: 600,
                  color: undefined,
                  offsetY: -10
                },
                value: {
                  show: true,
                  fontSize: '14px',
                  fontFamily: undefined,
                  fontWeight: 400,
                  color: undefined,
                  offsetY: 16,
                  formatter: function (val) {
                    return val + '%'
                  }
                },
                total: {
                  show: false,
                  label: 'Total',
                  color: '#373d3f',
                  fontSize: '16px',
                  fontFamily: undefined,
                  fontWeight: 600,
                  formatter: function (w) {
                    return w.globals.seriesTotals.reduce((a, b) => {
                      return a + b
                    }, 0) / w.globals.series.length + '%'
                  }
                }
          }
      }
  }`

**Comment by RomRider** [comment]
No, because I need to retrieve it from the config and I don't know where to pick that from because I don't know from which serie it comes.
Also the value I retrieve is the value already converted to percentages and this is not what you will want to display.

I already did some tests, even using undocumented stuff, but it's not working.

**Comment by Hakzel** [comment]
I get your point now. Thought it would be possible for single series radialBars to swap the percentage to another unit and convert the 'x' back to it's original value using the 'min' and 'max' attributes. 

But I now see why this can't be done so easily as I thought 

Nonetheless, I highly appreciate your work! :-)

**Comment by RomRider** [comment]
Thanks 🙂 

I've opened a full feature request, let's see if it gets through at some point:
https://github.com/apexcharts/apexcharts.js/issues/2216

**Comment by sorentorp** [comment]
@Hakzel can you share you code for making these guages with apexcharts?
